back/engines/commercial/insights/spec/acceptance/views_spec.rb

```diff
@@ -117,6 +122,19 @@
         expect(json_response).to match(expected_response)
       end
 
+      example_request 'copies topic to assignments' do
```

Comment:
```suggestion
      example 'copies topic to assignments', document: false do
        do_request
```
(bc `document: false` does not work with `example_request`)